domains/auth/Auth.API/API/Controllers/AuthController.cs

```diff
@@ -39,6 +41,19 @@ public AuthController(ILogger<AuthController> logger, IOidcProviders oidcProvide
         return _oidcProviders.CreateAuthorizationUri(state);
     }
 
+    [HttpPost]
+    [Route("/invalidate")]
+    public IActionResult Invalidate([Required] AuthState state)
+    {
+        if (state.IdToken.IsNullOrEmpty())
+        {
+            return BadRequest(nameof(state.IdToken) + " must not be null");
+        }
+
+        _oidcProviders.Logout(state.IdToken);
+        return Ok();
+    }
```

Comment:
The AuthState comes in as a encrypted string.
You need to  call decrypt it using the ICryptographyService.Decrypt, and then deserilize it into the AuthState Object

Reply:
Fixed in 21a94a8